src/Hamiltonians/abstract.jl

```diff
@@ -232,6 +232,69 @@ function build_sparse_matrix_from_LO(
     end
 end
 
+"""
+    basis = build_basis_only_from_LO(
+        ham, add; cutoff, filter=nothing, sort=false, kwargs...
+    )
+
+Get a vector of all basis element of a linear operator `ham` that are reachable (via 
+non-zero matrix elements) from the address `add`. Does not compute or return the matrix.
```

Comment:
Maybe here already one could refer to `BasisSetRep`, which computes and returns the matrix.